metrics/import_host_metrics/src/metrics/fields.rs

```diff
@@ -0,0 +1,590 @@
+use influxdb::WriteQuery;
```

Comment:
I've restored the census of all the fields. Unfortunately it is necessary, because we need always to know whether it's an integer or a float for influx.

I've tried to do a as_u64 and as_f64 on json values, but it doesn't work well, because if a float has not decimal when parsed, it is treated as an integer and influx fails when you import different types for the same values.

So probably the best way is to do like this.